.github/copilot-instructions.md

```diff
@@ -0,0 +1,214 @@
+# MvvmCross Development Instructions
+
+MvvmCross is a cross-platform MVVM framework for .NET supporting Android, iOS, MacCatalyst, TvOS, macOS, WinUI, and WPF. This guide provides essential commands and workflows for GitHub Copilot to work effectively with the codebase.
+
+**Always reference these instructions first and fallback to search or bash commands only when you encounter unexpected information that does not match the info here.**
+
+## Platform Requirements
+
+**CRITICAL PLATFORM LIMITATION:** All MvvmCross projects require iOS workloads even for Android and Core development due to multi-targeting. **Full builds and tests require Windows or macOS.**
+
+### System Requirements
+
+**Required**
+- .NET 9.0.100 SDK (specified in global.json)
```

Comment:
We are using latest .NET SDK 9.0.304 as defined in https://github.com/MvvmCross/MvvmCross/blob/develop/.github/actions/shared/action.yml#L7